subsys/shell/shell.c

```diff
@@ -1175,7 +1175,7 @@ void shell_thread(void *shell_handle, void *arg_log_backend,
 				  &shell->ctx->signals[i]);
 	}
 
-	err = shell_start(shell);
+	err = shell->iface->api->enable(shell->iface, false);
```

Comment:
What is the difference between this enable and start? Why these were separated?

Reply:
shell_start has been moved not replaced. `enable` is part of transport interface